src/styles/blocks/page.scss

```
@@ -0,0 +1,30 @@
+html {
+  font-size: 10px;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
```

only box-sizing can be added to `*{ }` block 
```suggestion
* {
  box-sizing: border-box;
}
```